docs/_docs/installation.md

```diff
@@ -21,6 +21,7 @@ requirements you’ll need to make sure your system has before you start.
 - [NodeJS](https://nodejs.org/), or another JavaScript runtime (Jekyll 2 and
 earlier, for CoffeeScript support).
 - [Python 2.7](https://www.python.org/downloads/) (for Jekyll 2 and earlier)
+- [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn't have them installed)
```

Comment:
@parkr Yes, that seems great. Also, gcc -v and make -v work as well, so we can use the shorter versions.

What syntax do you suggest for adding the information? A note, a code block or just inline code?